stanford/stage-ec2.ini

```diff
@@ -1,8 +0,0 @@
-[ec2]
-regions=us-west-1
-regions_exclude = us-gov-west-1
-destination_variable=public_dns_name
-vpc_destination_variable=private_dns_name
-cache_path=ec2_cache/stage
```

Comment:
This was the only unique line in the file.
I haven't yet encountered an issue sharing a common `cache_path`.
If anything, I would expect a resolution conflict between the `prod` and `stage` VPCs.
We should confirm this.